pkg/client/event.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"context"
+
+	apiv1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
+)
+
+func (c *DefaultClient) EventStream(ctx context.Context, opts *EventStreamOptions) (<-chan apiv1.Event, error) {
```

Comment:
I opted for returning a channel so we can implement watching later without needing a separate method.